src/bls.mjs

```diff
@@ -1312,31 +1004,24 @@ async function gatherSigningStartImpl( optsSignOperation ) {
                     optsSignOperation.jarrNodes.length ) {
                 clearInterval( iv );
                 optsSignOperation.fn(
-                    "signature error(2), got " +
-                    optsSignOperation.joGatheringTracker.nCountErrors +
-                    " errors(s) for " + optsSignOperation.jarrNodes.length +
-                    " node(s)", optsSignOperation.jarrMessages,
-                    null
+                    `signature error(2), got ${optsSignOperation.joGatheringTracker.nCountErrors}` +
+                    ` errors(s) for ${optsSignOperation.jarrNodes.length} node(s)`,
+                    optsSignOperation.jarrMessages, null
```

Comment:
Do we need to pass null?

Reply:
`null` indicates existence of function parameter

Reply:
Removed.